.github/workflows/lint-pr.yml

```diff
@@ -0,0 +1,13 @@
+name: Lint PR
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+  merge_group:
```

Comment:
Do we need `merge_group` here as well? Can the PR title change while the PR is in the merge queue?

Reply:
merge queue requires all `required` checks in pull request to also pass in merge queue, so we need to call that workflow in merge queue even though it actually doesn't check for anything.

from the docs: 
> Merge queue and pull requests checks are coupled and configured under branch protection rules or rulesets.

https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue#triggering-merge-group-checks-with-github-actions